.github/workflows/bench.yml

```diff
@@ -45,6 +52,28 @@ jobs:
         cabal-version: ${{ matrix.cabal }}
         enable-stack: false
 
+    - name: Platform config
+      run: |
+        echo "CABAL_STORE_DIR=~/.cabal/store" >> $GITHUB_ENV
+        echo "CABAL_PKGS_DIR=~/.cabal/packages" >> $GITHUB_ENV
+
+    # Needs to be before Cache Cabal so the cache can detect changes to the modified cabal.project file
+    - if: matrix.ghc == '9.0.1'
```

Comment:
this section could be removed

Reply:
Ok. Wanted to keep it so people would not forget it on the next update.